### DISCUSSION TITLE
Is there a Community Discord or Forums

### DISCUSSION THREAD

**Discussion by AmeyaGawde326** [Docile-Alligator/Infinity-For-Reddit#1127] [General]
I want to contribute towards this project i dont have much experience in opensource and the code base is extremely huge so being in a community like Discord or Forums would have helped a lot 

Is there something like that for Infinity

**Comment by Wladefant** [comment]
Hey,
Thank you for your (future 😅) support. We have a subreddit and github discussions that should work like a forum. 

Additionally soon (hopefully) I will write a documentation  so that the contribution will be easier
